yt_dlp/extractor/playsuisse.py

```diff
@@ -120,6 +120,16 @@ class PlaySuisseIE(InfoExtractor):
             id
             name
             description
+            descriptionLong
+            year
+            contentTypes
+            countries
+            composers
+            directors
+            writers
+            mainCast
+            productionCompanies
+            productionCountries
```

Comment:
yt-dlp's graphql queries are intentionally kept as lean as possible. Don't add fields we aren't using in the extractor code
```suggestion
            descriptionLong
            year
            contentTypes
            directors
            mainCast
            productionCountries
```